src/com/itacademy/eshop/product/Description.java

```diff
@@ -0,0 +1,31 @@
+package com.itacademy.eshop.product;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Description {
+
+    private String productDescription;
+    private String descriptionAdditionTime;
+
+    public Description() {}
+
+    public String getProductDescription() {
+        return productDescription;
+    }
+
+    public void setProductDescription(String productDescription) {
+        this.productDescription = productDescription;
+    }
+
+    public String getDescriptionAdditionTime() {
+        return descriptionAdditionTime;
+    }
+
+    public void setDescriptionAdditionTime() {
```

Comment:
It is good to set a date when the description was added. But to have it more elegant you can instantiate the time when you will create your description object in the constructor like:
`public Description(String description) {
this.productDescription = description;
this.descriptionAdditionTime = LocalDateTime.now();
}`